python/cudf/cudf/core/indexed_frame.py

```diff
@@ -3936,7 +3936,7 @@ def round(self, decimals=0, how="half_even"):
 
         cols = (
             col.round(decimals[name], how=how)
-            if name in decimals and col.dtype.kind in "fiu"
+            if name in decimals and (col.dtype.kind in "fiu" or isinstance(col.dtype, (cudf.Decimal32Dtype, cudf.Decimal64Dtype)))
```

Comment:
Could you include `Decimal128Dtype` type too?

Reply:
Decimal128Dtype cannot be included because libcudf does not currently support it.
For reference, please see the libcudf documentation:
https://docs.rapids.ai/api/cudf/nightly/libcudf_docs/api_docs/transformation_unaryops/#_CPPv45roundRK11column_view